src/test/kotlin/blackjack/domain/CardNumberCalculatorTest.kt

```diff
@@ -52,7 +51,7 @@ class CardNumberCalculatorTest {
     }
 
     @ParameterizedTest
-    @ValueSource(strings = ["ONE", "TWO", "THREE", "FOUR", "FIVE", "SIX", "SEVEN", "EIGHT", "NINE", "TEN"])
+    @ValueSource(strings = ["TWO", "THREE", "FOUR", "FIVE", "SIX", "SEVEN", "EIGHT", "NINE", "TEN"])
```

Comment:
```suggestion
    @ValueSource(strings = ["A", "TWO", "THREE", "FOUR", "FIVE", "SIX", "SEVEN", "EIGHT", "NINE", "TEN"])
```
일반적으로 Ace도 숫자 계산은 카드 숫자를 기본으로 합니다! 💪

Reply:
요 부분은 조금 궁금한것이 있어서 질문 남겨봅니다~

요구사항에서 "예외로 Ace는 1 또는 11로 계산할 수 있으며" 라는 문구로 다른 숫자들과는 별개로 특수한 케이스로 지정을 하였는데요.
저는 이것을 구현할때 생각했던것으로

플레이어가 뽑은 카드가 ACE 일 때,
기존의 카드의 숫자합과 ACE의 숫자합이 21보다 작거나 같으면
"21에 최대한 가깝게 만들고 그 다음 카드를 안뽑으면 그만일 것! " 이라는 생각을 하지 않을까? 
라는 의도로 

새로 A를  뽑았을 때 "기존 숫자의 카드 합 + ACE <= 21" => (ACE = 11 )
새로 A를  뽑았을 때 "기존 숫자의 카드 합 + ACE > 21" => (ACE = 1 )

이런 사고의 흐름으로 계산 로직을 만들어보았습니다.

그래서 기존에는 A는 다른 숫자들과는 별개의 특수한 상황으로 보는것이 맞는 것 같아
 숫자들과는 구분지어서 테스트를 작성하였는데요~

리뷰어님의 의견을 들어보니 ACE도 숫자 계산을 해야하는군요!
이 부분도 일종의 테스트하기 어려운 로직인 것 같아 전략패턴을 적용해서 고쳐보았습니다.

서론이 길었는데요. 그래서 질문이 무엇인지 말씀을 드리면~!

요구사항에는 구체적인 명시가 없이 "1또는 11로 처리할 수 있다." 라는 말만 적혀있어서 개발자 입장에서 애매해지는 부분이 되는것 같아요.
기능 요구사항은 A를 만들어오라고 했는데 저는 A'을 만들어 버린 것일까요? 😅

리뷰어님은 ACE를 1 또는 11로 처리하실때 어떤식으로 구현하셨는지 궁금하네요! 
그리고 이런 요구사항이 애매한 상황에서 어떤식으로 대처하시는지 리뷰어님의 경험도 궁금합니다!

~~(질문 요구사항이 너무 많은가요? 🥲)~~

Reply:
아마 블랙잭 게임 룰이 복잡해서 그런 것 같은데요. 😅 
현실 세계에서 게임을 한다고 가정해보면 많은 도움이 되더라구요.
그리고 객체는 서로 협력 관계임을 인지하고 계시면 좋을 것 같습니다.
플레이어마다 자신이 뽑은 카드 리스트를 가지고 있지만, 플레이어 스스로가 카드 점수를 알지는 못합니다. (블랙잭 게임을 모르는 사람도 있잖아요?)
그렇다면 플레이어는 언제 카드 점수를 알 수 있을까요?
보유한 카드들의 점수를 공개해야할 때이지 않을까요?
그 때에는 플레이어 입장에서 "내가 가지고 있는 카드 리스트를 알려줄테니 카드 점수가 몇점인지 알려줘" 라고 누군가(객체)에게 메시지를 보냅니다.
그러면 그 객체는 카드리스트만 보고 점수를 구해주면 됩니다.
바로 이 시점에 ACE가 1점인지 11점인지 판단해서 계산해주면 되지 않을까요? 😄 

> 그리고 이런 요구사항이 애매한 상황에서 어떤식으로 대처하시는지 리뷰어님의 경험도 궁금합니다!

저는 해당 기능이 어느 시점에 필요한지를 파악하고, 그 후에 구조를 잡는 것 같습니다. 😅 
이 부분은 시행착오를 겪으시면 될 것 같습니다!